cli/azd/cmd/testdata/TestUsage-azd-mcp.snap

```diff
@@ -1,8 +1,12 @@
 
-
+Manage Model Context Protocol (MCP) server.
```

Comment:
If `azd mcp` commands are alpha should we include "(Alpha)" in the descriptions? I believe we did something similar for `azd add` when composability was in alpha.